lib/turn_junebug_expressway/message_engine.ex

```diff
@@ -127,8 +137,10 @@ defmodule TurnJunebugExpressway.MessageEngine do
           Basic.reject(channel, tag, requeue: not redelivered)
       end
   rescue
-    _exception ->
+    exception ->
       :ok = Basic.reject(channel, tag, requeue: not redelivered)
-      IO.puts("Error with #{payload}")
+      IO.puts("Error with event #{payload}")
+      # credo:disable-for-next-line
+      IO.inspect(exception)
```

Comment:
Leaving some debugging things in here. Planning to clean up at a later stage when everything is working.